src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -143,6 +147,8 @@ namespace winrt::TerminalApp::implementation
     DEFINE_EVENT(AppKeyBindings, DecreaseFontSize,  _DecreaseFontSizeHandlers,  TerminalApp::DecreaseFontSizeEventArgs);
     DEFINE_EVENT(AppKeyBindings, ScrollUp,          _ScrollUpHandlers,          TerminalApp::ScrollUpEventArgs);
     DEFINE_EVENT(AppKeyBindings, ScrollDown,        _ScrollDownHandlers,        TerminalApp::ScrollDownEventArgs);
+    DEFINE_EVENT(AppKeyBindings, ScrollUpPage,		_ScrollUpPageHandlers,		TerminalApp::ScrollUpPageEventArgs);
```

Comment:
spacing between `ScrollUpPage,		_ScrollUpPageHandlers` looks off compared to other lines